xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecPassthrough.cpp

```diff
@@ -61,6 +66,7 @@ bool CDVDAudioCodecPassthrough::Open(CDVDStreamInfo &hints, CDVDCodecOptions &op
 
     case CAEStreamInfo::STREAM_TYPE_TRUEHD:
       m_codecName = "pt-truehd";
+      CheckOutputDeviceRAW();
```

Comment:
(Mis)Using the hints might be better suited.

Reply:
used  `CDVDCodecOptions &options`